test/e2e_node/node_problem_detector_linux.go

```diff
@@ -237,9 +233,19 @@ var _ = framework.KubeDescribe("NodeProblemDetector [NodeFeature:NodeProblemDete
 					},
 					Containers: []v1.Container{
 						{
-							Name:    name,
-							Image:   image,
-							Command: []string{"sh", "-c", "touch " + logFile + " && /node-problem-detector --logtostderr --system-log-monitors=" + configFile + fmt.Sprintf(" --apiserver-override=%s?inClusterConfig=true", framework.TestContext.Host)},
+							Name:  name,
+							Image: image,
+							Command: []string{
+								"sh",
+								"-c",
+								// `ServiceAccount` admission controller is disabled in node e2e tests, so we could not use
+								// inClusterConfig here.
+								fmt.Sprintf("touch %s && /node-problem-detector --logtostderr --system-log-monitors=%s --apiserver-override=%s?inClusterConfig=false&auth=%s",
```

Comment:
Actually, is the not found error related to putting those into one element in the string slice? Should we break them done into multiple elements, each without any space in it?

Reply:
Never mind, there is sh -c. Just confused about the behavior change on file not found.